cmd/cluster/cluster_create.go

```diff
@@ -0,0 +1,252 @@
+package cluster
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/armory/armory-cli/cmd/agent"
+	"github.com/armory/armory-cli/cmd/login"
+	"github.com/armory/armory-cli/pkg/config"
+	"github.com/armory/armory-cli/pkg/configuration"
+	errorUtils "github.com/armory/armory-cli/pkg/errors"
+	"github.com/armory/armory-cli/pkg/model"
+	"github.com/armory/armory-cli/pkg/output"
+	"github.com/samber/lo"
+	"github.com/schollz/progressbar/v3"
+	"github.com/spf13/cobra"
+	"io/ioutil"
+	"math/rand"
+	"os"
+	"time"
+)
+
+const (
+	sandboxFilePath    = "/.armory/sandbox"
+	charset            = "abcdefghijklmnopqrstuvwxyz0123456789"
+	createClusterShort = "Creates a temporary kubernetes cluster"
+	createClusterLong  = "Creates a temporary kubernetes cluster for demo purposes. The created cluster is helpful for evaluating CD-as-a-Service and will be \n\n" +
```

Comment:
```suggestion
	createClusterLong  = "Creates a temporary kubernetes cluster for demo purposes. The created cluster is helpful for evaluating CD-as-a-Service and will be \n" +
```

the double newline looked strange to me.